src/data/equiv/basic.lean

```diff
@@ -350,6 +350,12 @@ end perm_congr
 protected lemma image_eq_preimage {α β} (e : α ≃ β) (s : set α) : e '' s = e.symm ⁻¹' s :=
 set.ext $ assume x, set.mem_image_iff_of_inverse e.left_inv e.right_inv
 
+@[simp] lemma symm_preimage_eq_image {α β} (e : α ≃ β) (s : set α) : e.symm ⁻¹' s = e '' s :=
+(e.image_eq_preimage s).symm
+
+@[simp] lemma symm_image_eq_preimage {α β} (e : α ≃ β) (s : set β) : e.symm '' s = e ⁻¹' s :=
+by conv_lhs {rw [←symm_preimage_eq_image, symm_symm e]}
+
```

Comment:
@gebner, we discussed these lemmas and whether they should be `simp` in another PR already, right?